### PR TITLE
Add new guide about signing transactions offline - Closes #1164

### LINKED ISSUE #1164: Create new guide: How to create and sign transactions offline
A new guide that explains how to sign transactions in a complete offline environment.

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1164

### CODE REVIEW

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc [hunk 1]
```diff
@@ -0,0 +1,321 @@
+= Creating transactions and signing them offline
+:url_transfer_asset: token-module.adoc#transferasset
+:url_register_multisig_asset: keys-module.adoc#registerasset
+:url_register_delegate_asset: dpos-module.adoc#registertransactionasset
+:url_vote_delegate_asset: dpos-module.adoc#votetransactionasset
+:url_unlock_asset: dpos-module.adoc#unlocktransactionasset
+:url_pom_asset: dpos-module.adoc#pomtransactionasset
+
+How to create and sign transaction in a complete offline environment.
```
Comment (ManuGowda) [inline modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc]:
```suggestion
How to create and sign a transaction in a completely offline environment.
```

**Review by ManuGowda — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+  },
+  signatures: [
+    <Buffer 3c 77 8c e7 b9 8e 72 e6 6b e1 83 86 b4 c1 97 b0 79 3d dc 33 ac ad 8d df 38 d3 52 9f 6a 76 ba 5e 5a ed 54 22 3f b8 36 81 61 b0 2c 71 68 88 3b 09 df b3 ... 14 more bytes>
+  ],
+  id: <Buffer 95 d2 d3 29 90 cd c7 f3 ae e5 54 b3 f5 23 7b fb f3 4c 33 48 e5 83 72 7a ce dd e5 b3 b6 e3 e7 25>
+}
+*/
+----
+
+== Option B: Using the CLI to create a transaction
```
Comment (shuse2) [inline modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc:202]: Why is this section using both CLI and the code? maybe if it's only saying CLI, then finish with CLI would be good

**Review by shuse2 — COMMENTED** [review]
Option A looks good =)

File: modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc (hunk 2 above)
Comment (Tschakki) [reply modules/ROOT/pages/guides/node-management/signing-transactions-offline.adoc:202]: Ah I didn't check that you can also sign tx offline with CLI. But can you give me a working example? I tried it offline but received the following error: 

```
$ ./bin/run transaction:sign --offline 0802100018012080ade2042a20840c66741a76f936bed0a4c308e4f670156e1e1f6b91640bb8d3dd0ae2b3581e32290814121485c12d39041bc09e1f89dfeffe4b87cfcfe79fb21a0f486170707920626972746864617921
    TypeError: The "path" argument must be of type string. Received undefined
    Code: ERR_INVALID_ARG_TYPE
```

**Review by shuse2 — APPROVED** [review]

**Pull request merged**